posthog/settings/web.py

```diff
@@ -68,10 +67,9 @@
     MIDDLEWARE.append("django_statsd.middleware.StatsdMiddlewareTimer")
 
 # Append Enterprise Edition as an app if available
-if EE_AVAILABLE:
```

Comment:
I thought EE_AVAILABLE is meant to check if ee code is available not necessarily if a license exists. If we enable clickhouse by default in this PR without moving analytics code around, we're also going to need EE code always available. Paid features still have to be locked by a license check

Reply:
That's true practically, but not from a licensing perspective. The `ee/` directory is the only one with a proprietary license, so we will definitely need to move ClickHouse code out of it for CH to actually be FOSS.

Reply:
Ohh yeah for FOSS purposes definitely. That's why I'm trying to stop the FOSS syncing for a brief period until we move that code otherwise this PR would turn into too many lines more than it already is